docs/public-networks/reference/cli/options.md

```diff
@@ -724,23 +724,16 @@ Contact email address to send to the Ethstats server specified by [`--ethstats`]
     fast-sync-min-peers=8
     ```
 
-The minimum number of peers required before starting [fast synchronization](../../get-started/connect/sync-node.md#fast-synchronization).
+The minimum number of peers required before starting [fast synchronization](../../get-started/connect/sync-node.md#fast-synchronization)
+in [proof of work](../../how-to/use-pow/mining.md) networks.
 The default is 1.
 
-!!! note
+!!! important
 
-    If synchronizing in `FAST` mode, most historical world state data is unavailable. Any methods
-    attempting to access unavailable world state data return `null`.
+    This option only applies to proof of work networks.
 
 ### `genesis-file`
 
-Use the genesis file to create a custom network.
```

Comment:
the changes to `genesis-file` section are unrelated, have they been inserted by error?

Reply:
No, I noticed that the description in of `genesis-file` was redundant and mis-formatted, so I fixed it in this PR as well (as per the PR description).